src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i >= figures.length / 2) {
+                figures[i] = figureSupplier.getDefaultFigure();
+            } else {
+                figures[i] = figureSupplier.getRandomFigure();
+            }
+        }
+
+        for (Figure fig : figures) {
```

Comment:
dont obfuscate your code :)
```suggestion
        for (Figure figure : figures) {
```